powa_collector/powa_worker.py

```diff
@@ -74,6 +79,10 @@ def __get_powa_version(self, conn):
         return res
 
     def __maybe_load_powa(self, conn):
+        """Loads Powa if it's not already and it's possible.
+        We need to have a 4.0+ extension, and we can load it on the fly
```

Comment:
The load is not "if possible" but "if required".

It should state that only powa 4+ is supported, and that only 4.0.x requires an explicit LOAD (version 4.1+ handles the lack of GUC).